README.md

```diff
@@ -1,5 +1,7 @@
+# ⚠️ Work in progress
+If you just discovered this repo, **please be patient**. There will be expected issues and also the information down below will not be fully accurate until we have finished fizzyfacepunch. **Thank you!**
+
 # FizzyFacepunch
-Project Website: https://github.com/Chykary/FizzyFacepunch
```

Comment:
I removed this since it added no benefit information regarding the readme.md and not to mention it was pointing to the github repo. which the user would already be on.